airflow/providers/amazon/aws/example_dags/example_eks_using_defaults.py

```diff
@@ -73,7 +72,7 @@
         task_id="run_pod",
         cluster_name=CLUSTER_NAME,
         image="amazon/aws-cli:latest",
-        cmds=["sh", "-c", "ls"],
+        cmds=["sh", "-c", "echo Test Airflow; date"],
```

Comment:
Purely out of curiosity and so I know for he future, is there a reason for this change or just preference??

Reply:
`ls` was returning empty output, which could look like a problem. Now we have a user-friendly message.

Reply:
Got it, I'll keep that in mind for the future, thanks.

Reply:
Nice change indeed.